modules/xfem/doc/content/source/userobjects/NodeValueAtXFEMInterface.md

```diff
@@ -0,0 +1,19 @@
+# NodeValueAtXFEMInterface
+
+!syntax description /UserObjects/NodeValueAtXFEMInterface
+
+## Overview
+
+The values of field variables on either side of a moving interface are often needed to define the interface velocity. These variables and their gradients might be discontinuous across the interface. The fact that the interface does not lie on the standard quadrature points makes it difficult to get the quantities on the interface using existing functions in MOOSE.
+
+`NodeValueAtXFEMInterface` provides functions to return the value of a field variable and its gradient on the positive and negative sides of the interface. The `NodeValueAtXFEMInterface` object is used by the `XFEMMovingInterfaceVelocityBase` derived class to calculate the velocity of the interface for moving interface problems.
```

Comment:
Can you add one or two sentences just to briefly describe how the values (and gradients) are calculated? I think that would be very helpful.

Reply:
@bwspenc yes, I replaced `PointValue` with `NodeValue` just because the values are evaluated at each cut mesh node. Maybe we can further discuss it.

@gary added.